app/services/ncr/work_order_value_normalizer.rb

```diff
@@ -41,6 +42,12 @@ def normalize_soc_code
       end
     end
 
+    def normalize_approving_official_email
+      if !work_order.approving_official_email.present? && work_order.approvers.any?
```

Comment:
this might be a little more readable: `if work_order.approving_official_email.blank? && work_order.approvers.any?`